deployment/helm/values.yaml

```diff
@@ -0,0 +1,138 @@
+# Default values for open-zaak.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+settings:
+  name: openzaak
+  subdomain: openzaak
+  domain: zuid-drecht.nl
+  env: dev
+  module: openzaak.conf.docker
+  secret: 7y82z1%ef)k@f=u-_%ri71lk4=7jy^z5#@6d4*92=t@p0-yk5k
+  dsn:
+  version: latest
+  pullPolicy: Always
+  debug: 1
```

Comment:
default values should be generic (e.g. use `gemeente.nl` instead of real domain names) _and_ default to secure values -> `debug: 0`, `version` pinned to a particular version (it will happen that `latest` is broken and if your cluster _then_ pulls a new version, that's going to be annoying)

Definitely don't put realistic secret keys in example values!